packages/synapse-react-client/src/assets/icons/FileWithShield.tsx

```diff
@@ -0,0 +1,12 @@
+import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon'
+import { ReactComponent as Svg } from './FileWithShield.svg'
```

Comment:
This is the icon that Adam is currently using to represent RecordSets. I don't know that it's the most appropriate, but it works for now.